lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,8 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
I think the two should be used in conjunction with each other: general fortran flags (e.g., if you need to pass something to *all* fortran compilations), and per-bindings flags (e.g., if you need to pass something to just the `use mpi_f08` tests, as this PR was created for).